plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObjectPdo.class.php

```diff
@@ -945,286 +945,31 @@ protected function getPhysicalObjects()
-          {
-            $event = array();
-
-            $item->registerXPathNamespace('s', 'info:lc/xmlns/premis-v2');
-
-            if (0 < count($value = $item->xpath('s:eventType')))
-            {
-              $event['type'] = (string)$value[0];
-            }
-
-            if (0 < count($value = $item->xpath('s:eventDateTime')))
-            {
-              $event['dateTime'] = arElasticSearchPluginUtil::convertDate((string)$value[0]);
-            }
-
-            if (0 < count($value = $item->xpath('s:eventDetail')))
-            {
-              $event['detail'] = (string)$value[0];
-            }
-
-            if (0 < count($value = $item->xpath('s:eventOutcomeInformation/s:eventOutcome')))
-            {
-              $event['outcome'] = (string)$value[0];
-            }
-
-            if (0 < count($value = $item->xpath('s:eventOutcomeInformation/s:eventOutcomeDetail/s:eventOutcomeDetailNote')))
-            {
-              $event['outcomeDetailNote'] = (string)$value[0];
-            }
-
-            foreach ($item->xpath('s:linkingAgentIdentifier') as $linkingAgent)
-            {
-              $linkingAgentIdentifier = array();
-
-              $linkingAgent->registerXPathNamespace('s', 'info:lc/xmlns/premis-v2');
-
-              if (0 < count($value = $linkingAgent->xpath('s:linkingAgentIdentifierType')))
-              {
-                $linkingAgentIdentifier['type'] = (string)$value[0];
-              }
-
-              if (0 < count($value = $linkingAgent->xpath('s:linkingAgentIdentifierValue')))
-              {
-                $linkingAgentIdentifier['value'] = (string)$value[0];
-              }
-
-              $event['linkingAgentIdentifier'][] = $linkingAgentIdentifier;
-            }
-
-            $metsData['event'][] = $event;
-          }
-
-          // Agents
-          foreach ($document->xpath('//m:amdSec[@ID="'.(string)$amdSecId.'"]/m:digiprovMD/m:mdWrap[@MDTYPE="PREMIS:AGENT"]/m:xmlData/m:agent') as $item)
-          {
-            $agent = array();
-
-            $item->registerXPathNamespace('m', 'http://www.loc.gov/METS/');
-
-            if (0 < count($value = $item->xpath('m:agentIdentifier/m:agentIdentifierType')))
-            {
-              $agent['identifierType'] = (string)$value[0];
-            }
-
-            if (0 < count($value = $item->xpath('m:agentIdentifier/m:agentIdentifierValue')))
-            {
-              $agent['identifierValue'] = (string)$value[0];
-            }
-
-            if (0 < count($value = $item->xpath('m:agentName')))
-            {
-              $agent['name'] = (string)$value[0];
-            }
-
-            if (0 < count($value = $item->xpath('m:agentType')))
-            {
-              $agent['type'] = (string)$value[0];
-            }
+      return;
+    }
 
-            $metsData['agent'][] = $agent;
-          }
+    // Get METS filepath
+    $metsFilepath = sfConfig::get('sf_web_dir').
+      DIRECTORY_SEPARATOR.'uploads'.
```

Comment:
Use sf_uploads_dir, because uploads/ doesn't have to be necessarily located under the root dir.